cf_xarray/datasets.py

```diff
@@ -225,6 +225,10 @@ def _create_mollw_ds():
 def _create_inexact_bounds():
     # Dataset that creates rotated pole curvilinear coordinates with CF bounds in
     # counterclockwise order that have precision issues.
+    # dataset created using: https://gist.github.com/larsbuntemeyer/105d83c1eb39b1462150d3fabca0b66b
```

Comment:
Very nice. In a future PR, it would be nice to add this as a "demo" notebook which creates a datasets, and illustrates creation of bounds/ extracting grid_mapping etc.